synthtool/gcp/templates/python_library/CONTRIBUTING.rst

```diff
@@ -199,24 +216,24 @@ Supported Python Versions
 
 We support:
 
--  `Python 3.6`_
--  `Python 3.7`_
--  `Python 3.8`_
--  `Python 3.9`_
-
-.. _Python 3.6: https://docs.python.org/3.6/
-.. _Python 3.7: https://docs.python.org/3.7/
-.. _Python 3.8: https://docs.python.org/3.8/
-.. _Python 3.9: https://docs.python.org/3.9/
-
+{% for v in unit_test_python_versions -%}
+-  `Python {{ v -}}`_
+{% endfor %}
+{% for v in unit_test_python_versions -%}
+.. _Python {{ v -}}: https://docs.python.org/{{- v -}}/
+{% endfor %}
 
 Supported versions can be found in our ``noxfile.py`` `config`_.
 
 .. _config: https://github.com/{{ metadata['repo']['repo'] }}/blob/master/noxfile.py
 
 
+{% for v in unit_test_python_versions -%}
+  {% if loop.first -%}
 We also explicitly decided to support Python 3 beginning with version
-3.6. Reasons for this include:
+{{ v -}}. Reasons for this include:
```

Comment:
Can `Reasons for this include` be pulled outside of the for loop?

Reply:
Fixed in 462586d